src/PatientChecking/PatientChecking/Controllers/AppointmentController.cs

```diff
@@ -87,5 +91,31 @@ public async Task<IActionResult> Detail(int appointmentId)
             };
             return View(appointmentDetailViewModel);
         }
+
+        public IActionResult Update(AppointmentDetailViewModel appointmentDetailViewModel)
+        {
+            var appointment = new Appointment
+            {
+                AppointmentId = appointmentDetailViewModel.AppointmentId,
+                CheckInDate = DateTime.Parse(appointmentDetailViewModel.CheckInDate),
+                MedicalConcerns = appointmentDetailViewModel.MedicalConcerns,
+                PatientId = appointmentDetailViewModel.PatientId,
+                Status = appointmentDetailViewModel.Status
+            };
+            if(appointment.CheckInDate < DateTime.Now)
+            {
+                _notyf.Error("Update Appointment Failed!");
+            }
+            var result = _appointmentService.UpdateAppointment(appointment);
+            if(result > 0)
+            {
+                _notyf.Success("Update Appointment successfully!");
```

Comment:
Update Appointment **S**uccessfully

Reply:
I have removed this since I use ViewMessageHelper instead of INotyService. And I have capitalized the letter "S"